build/tools/SharedCodeGenerator/Program.cs

```diff
@@ -107,8 +107,11 @@ private static void GenerateSupportedPlatformsReadmeFile(string repoDir)
             {
                 sw.WriteLine("# Supported platforms and versions");
                 sw.WriteLine();
-                foreach (var subDirPath in Directory.GetDirectories(platformsDir))
+                string[] subDirs = Directory.GetDirectories(platformsDir);
+                string[] sortedSubDirs = subDirs.OrderBy(n => n).ToArray();
+                foreach (var subDirPath in sortedSubDirs)
                 {
+                    Console.WriteLine(subDirPath);
```

Comment:
Could you clarify if we need to keep the Console.WriteLine? It feels like if we need to keep it, then we should add more info in there to make the log more actionable.

Reply:
completely missed this one when cleaning up the code.. I'll delete that asap

Reply:
Thanks!